### DISCUSSION TITLE
Intallation error

### DISCUSSION THREAD

**Discussion by Rohit-Raut** [mctools/ncrystal#132] [General]
I am new to Mcstas and trying to download, but I am getting following error. Could someone help me out please:
/bin/sh: 1: BISON_EXECUTABLE-NOTFOUND: not found
make[2]: *** [CMakeFiles/mcstas.dir/build.make:79: work/src/instrument.tab.h] Error 127
make[1]: *** [CMakeFiles/Makefile2:82: CMakeFiles/mcstas.dir/all] Error 2
make: *** [Makefile:156: all] Error 2

Any help is really appreciated

**Comment by tkittel** [comment]
Dear Rohit,

That sounds like you have posted in the wrong issue tracker. This is the NCrystal tracker, the one for McStas is at: https://github.com/McStasMcXtrace/McCode/issues

Cheers,
Thomas
